### PR TITLE
[ci][tvmbot] Fix spelling error, metionable -> mentionable

### PR DESCRIPTION
**Pull request by bkmgit**
In AUTH_CHECKS, "mentionable_users" was "metionable_users".
This pull request fixes this spelling error.

### CODE REVIEW

**Review by Mousius — APPROVED** [review]

**Review by Mousius — CHANGES_REQUESTED** [review]
There's a line below in this file:

```
    auth = [AUTH_CHECKS["metionable_users"]]
```

Can you update there as well and add a test as this almost got broken 😿 

@driazati ^^^

**Comment by driazati** [comment]
also fyi @Mousius until #12941 merges these files aren't actually used in CI, their `main` counterparts are

**Comment by bkmgit** [comment]
Thanks for catching that. Thinking about appropriate tests.

**Comment by driazati** [comment]
Hey @bkmgit, we recently merged a PR #13276 that makes it so you can iterate and test on this PR without needing a committer in the loop, can you rebase this PR to fix the conflicts then we can get it merged? thanks!

**Review by driazati — APPROVED** [review]

**Comment by Mousius** [comment]
> Hey @bkmgit, we recently merged a PR #13276 that makes it so you can iterate and test on this PR without needing a committer in the loop, can you rebase this PR to fix the conflicts then we can get it merged? thanks!

I think you've linked back to this PR @driazati, which one did you mean to refer to?

**Comment by driazati** [comment]
Ah thanks for catching, I meant to link #13368

**Pull request merged**
